jupyterlab_latex/_version.py

```diff
@@ -1 +1,22 @@
-__version__ = '3.0.0'
+import json
+from pathlib import Path
+
+__all__ = ["__version__", "__js__"]
+
+def _fetchJS():
+    settings = Path(__file__).parent.resolve() / "labextension/package.json"
+
+    try:
+        with settings.open() as f:
+            return json.load(f)
+    except FileNotFoundError:
+        pass
+
+    raise FileNotFoundError(f"Could not find package.json at {settings!s}")
+
+
+def _fetchVersion():
+    return _fetchJS()["version"]
+
+__js__ = _fetchJS()
+__version__ = _fetchVersion()
```

Comment:
can just be `__js__['version']`: we're not beholden to anybody for these nominally-private APIs (which haven't been released!)